src/components/ScreenreaderOnly.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export default styled.span`
```

Comment:
I would recommend that you put this in GlobalStyles.js and set the class for the elements in question.